src/markdown-pages/add-ons/rook.md

```diff
@@ -30,8 +30,10 @@ flags-table
 
 ## Block Storage
 
-For Rook versions 1.4.3 and later, block storage is required.
-For Rook versions earlier than 1.4.3, block storage is recommended in production clusters. For disk requirements, see [Add-on Directory Disk Space Requirements](/docs/install-with-kurl/system-requirements/#add-on-directory-disk-space-requirements).
+Rook versions 1.4.3 and later require a dedicated block device attached to each node in the cluster.
+To meet this block storage requirement, you must add an unformatted disk that is used only for Rook to each node.
```

Comment:
```suggestion
Rock (any version) require a dedicated block device attached to each node in the cluster. To meet this block storage requirement, you must add an unformatted disk that is used only for Rook to each node.
```
What about we try to simplify it? Why we need to say version X and later when it is valid for all versions so far?

Reply:
How someone can `add an unformatted disk`?
Have we a k8s doc explaining it?
Could we link it here?

Reply:
@camilamacedo86 great idea to add a link to more information about how to add an unformatted disk. @ricardomaraschini, do you know if adding an unformatted disk is a fairly standard task that we can expect the reader to know how to do, or is it something that has to do with using Kubernetes specifically that we can fairly easily link to? 

And @camilamacedo86 for your other question: 
> Why we need to say version X and later when it is valid for all versions so far?

As I understand it, this requirement applies to version 1.4.3 _and_ all versions that are released later than 1.4.3. In order to express this idea simply and in fewer words, we say "version 1.4.3 and later". The other benefit of saying "and later" is that it does not become out of date. As in, anytime Rook releases a new version, we don't have to worry about updating the documentation (until we need to make a change to this requirement on our end). This reduces overhead, which also reduces the likelihood of the documentation becoming inaccurate.

Please let me know if this makes sense, or if I misunderstood your question!

Reply:
I see so, the requirement is not valid for versions lower than 1.4.3? 
@ricardomaraschini might be able to give a better hand.

HAve we none requirements for the releases bellow 1.4.3?

Reply:
@paigecalvert I don't think we need to go as far as explaining how to add an unformatted disk to a computer or VM. We can't possibly know where the customer is running and this process will be different depending on the type of cloud provider in use (assuming they are not running this on-premisse or even on bare metal).

Reply:
Sounds good @ricardomaraschini -- thanks for confirming! Will leave as is